README.md

```diff
@@ -173,6 +173,7 @@ Table of Contents
   * [Cloudmersive](https://cloudmersive.com/) — Utility API platform with full access to expansive API Library including Document Conversion, Virus Scanning, and more with 800 calls/month.
   * [Colaboratory](https://colab.research.google.com) — Free web-based Python notebook environment with Nvidia Tesla K80 GPU.
   * [Collect2](https://collect2.com) — Create an API endpoint to test, automate, and connect webhooks. Free plan allows for two datasets, 2000 records, 1 forwarder, and 1 alert.
+  * [CSV2GEO](https://csv2geo.com) — Creates a batch geocoding API endpoint with interactive map. Supports normal and reverse geocoding with rooftop results.  Coverage is worldwide and it provides daily free geocoding. 
```

Comment:
Can you please link to the free information? Nothing on their pricing page, only free thing is the input box on their front page which wouldn't really qualify for this list.